lisa/sut_orchestrator/qemu/schema.py

```diff
@@ -14,16 +14,35 @@ class CloudInitSchema:
     # Additional values to apply to the cloud-init user-data file.
     extra_user_data: Optional[str] = None
 
+@dataclass_json()
+@dataclass
+class LibvirtHost:
+    address: Optional[str] = None
+    username: Optional[str] = None
+    private_key_file: Optional[str] = None
+
+    # The directory where lisa will store VM related files (such as disk images).
+    # This directory must already exist and the test user should have write permission
+    # to it.
+    lisa_working_dir: Optional[str] = "/var/tmp"
```

Comment:
Is this value optional? Same thing for other variables.

Reply:
Updated.